src/modules/credentials/handlers/IssueCredentialHandler.ts

```diff
@@ -1,17 +1,58 @@
+import type { AgentConfig } from '../../../agent/AgentConfig'
 import type { Handler, HandlerInboundMessage } from '../../../agent/Handler'
+import type { CredentialRecord } from '../repository'
 import type { CredentialService } from '../services'
 
+import { createOutboundMessage } from '../../../agent/helpers'
+import { AriesFrameworkError } from '../../../error/AriesFrameworkError'
+import { AutoAcceptCredential } from '../../../types'
+import { CredentialUtils } from '../CredentialUtils'
 import { IssueCredentialMessage } from '../messages'
 
 export class IssueCredentialHandler implements Handler {
   private credentialService: CredentialService
+  private agentConfig: AgentConfig
   public supportedMessages = [IssueCredentialMessage]
 
-  public constructor(credentialService: CredentialService) {
+  public constructor(credentialService: CredentialService, agentConfig: AgentConfig) {
     this.credentialService = credentialService
+    this.agentConfig = agentConfig
   }
 
   public async handle(messageContext: HandlerInboundMessage<IssueCredentialHandler>) {
-    await this.credentialService.processCredential(messageContext)
+    const credentialRecord = await this.credentialService.processCredential(messageContext)
```

Comment:
Can you add some logging methods that explain what is happening with regards to auto accepting. (e.g. "auto sending ack with "always" policy" etc..)